isolated-contexts.bs

```diff
@@ -222,7 +227,9 @@ manipulation</dfn> if the following algorithm returns "`Sufficient`":
             *   |directive|'s [=directive/value=]'s [=set/size=] is 1
             *   |directive|'s [=directive/value=][0] is an
                 [=ASCII case-insensitive=] match for either the string
-                "`'none'`" or the string "`'self'`".
+                "`'<a href="https://w3c.github.io/webappsec-csp/#grammardef-none">none</a>'`"
+                or the string
+                "`'<a href="https://w3c.github.io/webappsec-csp/#grammardef-self">self</a>'`".
```

Comment:
Here and everywhere else, this incantation seems to work:

```suggestion
                "<a grammar>`'none'`</a>" or the string "<a grammar>`'self'`</a>".
```

Reply:
Thanks, I couldn't figure out how to get it to auto-link!